frontend/src/pages/Mesh/target/TargetPanelCommon.tsx

```diff
@@ -229,6 +229,7 @@ export const renderNodeLink = (meshData: MeshNodeData, style?: string): React.Re
       pfBadge = PFBadges.Workload;
       break;
     case MeshInfraType.WAYPOINT:
+    case MeshInfraType.ZTUNNEL:
```

Comment:
I think we would also need a Ztunnel option here: https://github.com/kiali/kiali/pull/8158/files#diff-c7d64567f73ef6022ea329b5714c80a862d754cbaa670ad16cd582dd52d66746L158 ? 
As the badge is showing unknown: 
![image](https://github.com/user-attachments/assets/917706d2-e5d8-4388-ae44-4bf744ab64a0)